analytic_code/README.rst

```diff
@@ -0,0 +1,17 @@
+analytic_code
+=============
+
+A module for companies who like to reference analytic account by their code.
+
+It has the following features:
+ * the analytic account names are displayed as "code - name"
+   (name_get)
+ * quick search on analytic account include code (name_search)
+
+Credits
+=======
+
+Contributors
+------------
+
+* Anthony Muschang <anthony.muschang@acsone.eu>
```

Comment:
You can add a maintainer section with the ACSONE logo. (https://github.com/acsone/acsone-addons/tree/8.0/multi_company_consolidation)